textproc/asciidoctorj/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           java 1.0
+
+github.setup        asciidoctor asciidoctorj 2.5.2 v
+github.tarball_from archive
+revision            0
+
+categories          textproc java
+platforms           darwin
+license             Apache-2
+supported_archs     noarch
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         \
+    AsciidoctorJ is the official library for running Asciidoctor on the JVM.
+long_description    {*}${description} Using AsciidoctorJ, you can convert \
+    AsciiDoc content or analyze the structure of a parsed AsciiDoc document \
+    from Java and other JVM languages.
+homepage            https://asciidoctor.org
+
+master_sites        https://repo1.maven.org/maven2/org/asciidoctor/asciidoctorj/${version}/
+distname            ${name}-${version}-bin
+worksrcdir          ${name}-${version}
+
+checksums           rmd160  ca537de047d7335b7475a6fb5392c3dda9c872c5 \
+                    sha256  889fd490a43e59e06c4f9d6692228e48ada2829acead7ea8c454a596c5cb2a96 \
+                    size    52364352
+
+use_zip             yes
+use_configure       no
+build {}
+
+patchfiles          patch-prefix-for-app-libs.diff
+
+destroot {
+    delete ${worksrcpath}/bin/${name}.bat
+
+    reinplace "s,@@PREFIX@@,${prefix}/lib/${name},g" ${worksrcpath}/bin/${name}
+    xinstall -m 0755 ${worksrcpath}/bin/${name} ${destroot}${prefix}/bin
+
+    xinstall -m 0755 -d ${destroot}${prefix}/lib/asciidoctorj
+    xinstall -m 0644 {*}[glob ${worksrcpath}/lib/*.jar] \
+        ${destroot}${prefix}/lib/asciidoctorj
+}
+
+livecheck.type      regex
+livecheck.url       https://repo1.maven.org/maven2/org/asciidoctor/asciidoctorj/
+livecheck.regex     (\\d+(?:\\.\\d+)*)/
```

Comment:
why not use the default from the github PG or is there a reason to override those?